tests/acceptance/features/bootstrap/WebUIAdminSharingSettingsContext.php

```diff
@@ -271,6 +278,46 @@ public function theAdministratorEnablesAddServerAutomatically($action) {
 		);
 	}
 
+	/**
+	 * @When the administrator adds :url as a trusted server using the webUI
+	 *
+	 * @param string $url
+	 *
+	 * @return void
+	 */
+	public function theAdministratorAddsAsATrustedServerUsingTheWebui($url) {
+		$this->adminSharingSettingsPage->addTrustedServer(
+			$this->getSession(),
+			$this->featureContext->substituteInLineCodes($url)
+		);
+	}
+
+	/**
+	 * @When the administrator deletes url :url from the trusted server list using the webUI
+	 *
+	 * @param string $url
+	 *
+	 * @return void
+	 */
+	public function theAdministratorDeletesAsATrustedServerUsingTheWebui($url) {
+		$this->adminSharingSettingsPage->deleteTrustedServer(
+			$this->getSession(),
+			$this->featureContext->substituteInLineCodes($url)
+		);
+	}
+
+	/**
+	 * @Then a error message for trusted server should contain :text
```

Comment:
Maybe:
```suggestion
	 * @Then a trusted server error message should be displayed on the webUI with the text :text
```

which is similar to `WebUIPersonalGeneralSettingsContext` 
```
Then a password error message should be displayed on the webUI with the text :wrongPasswordMessageText
```